### ISSUE TITLE
Method instance is lost when inflating a generic non-static method

### ISSUE THREAD

**Issue by kruvcraft21** [vfsfitvnm/frida-il2cpp-bridge#85]
Hi,
I have a method that I inflate

```ts
const obj =  ;//this Il2cpp Object
const c =  ;//this Il2cpp Class
obj.methods["Methods"].inflate(c).invoke<void>()
```
when I want to execute this code, I get a message
`[il2cpp] l ("Methods") is not static.`

how to call a generic method while having an object ?

**Comment by vfsfitvnm** [comment]
Hi!
The problem occurs [here](https://github.com/vfsfitvnm/frida-il2cpp-bridge/blob/master/src/il2cpp/structs/method.ts#L212) where I create a new `Il2Cpp.Method`, so it loses its instance object.
I didn't implement this one yet (should be super easy) because I couldn't find a method to test.

Would you tell me app name and class name? So I can implement this real quick

**Comment by kruvcraft21** [comment]
> Would you tell me app name and class name? So I can implement this real quick

Can I just write the class structure ? Or write to you personally somewhere?

**Comment by kruvcraft21** [comment]
> Would you tell me app name and class name? So I can implement this real quick

app name is War After and class name SignalBus

**Comment by vfsfitvnm** [comment]
Fixed in `0.6.7`, test snippet:
```ts
Il2Cpp.perform(() => { 
    const SystemString = Il2Cpp.Image.corlib.classes["System.String"];

    const AssemblyCSharp = Il2Cpp.Domain.assemblies["Assembly-CSharp-firstpass"].image;
    const SignalBus = AssemblyCSharp.classes["GameServerProtocol.Sources.SignalBus.SignalBus"];

    const signalBus = SignalBus.new();

    signalBus.methods.FireSignal.inflate(SystemString).invoke<void>(Il2Cpp.String.from("hello!"));

    console.log("Ok!");
});
```
